src/utils/framework-server.ts

```diff
@@ -38,12 +41,17 @@ export const startFrameworkServer = async function ({ cwd, settings }) {
     text: `Waiting for framework port ${settings.frameworkPort}. This can be configured using the 'targetPort' property in the netlify.toml`,
   })
 
+  if (settings.clearPublishDirectory && settings.dist) {
+    await rm(settings.dist, { recursive: true, force: true })
```

Comment:
this is the main change, all the rest is typescript conversion stuff

Reply:
haha I feel you just opened a just change types PR before performing the actual change